packages/cog/src/gdal/gdal.ts

```diff
@@ -54,12 +54,11 @@ export class GdalCogBuilder {
 
         this.config = {
             bbox: config.bbox,
-            alignmentLevels: config.alignmentLevels ?? 1,
-            compression: config.compression ?? 'webp',
-            resampling: config.resampling ?? 'bilinear',
-            blockSize: config.blockSize ?? 512,
+            alignmentLevels: config.alignmentLevels ?? GdalCogBuilderDefaults.alignmentLevels,
```

Comment:
it would be nice if we could just `{...GdalCogBuilderDefaults, ..config}` but config has empty keys so it doesnt work :'(